### PR TITLE
Added LiveStreamingMode parameter to New/Set-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/9175.

The instance parameter is not available anymore in Set-.

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -986,6 +985,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LiveStreamingMode
+Determines wheter you provide support for your users to stream their Teams meetings to large audiences through Real Time Messaging Protocol (RTMP).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:989]:
```suggestion
Determines whether you provide support for your users to stream their Teams meetings to large audiences through Real-Time Messaging Protocol (RTMP).
```

File: skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -1097,6 +1053,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LiveStreamingMode
+Determines wheter you provide support for your users to stream their Teams meetings to large audiences through Real Time Messaging Protocol (RTMP).
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md:1057]:
```suggestion
Determines whether you provide support for your users to stream their Teams meetings to large audiences through Real-Time Messaging Protocol (RTMP).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@dariomws

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**